Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsMatchingService.cs

```diff
@@ -145,5 +155,18 @@ private bool ComparePropertiesToArgsAndValues(IEnumerable<BotCommandProperty> pr
             }
             return true;
         }
+
+        private bool IsNumberValid(string value, Type type)
+        {
+            try
+            {
+                this._numberParser.Parse(value, type);
+                return true;
+            }
+            catch
+            {
+                return false;
```

Comment:
a co to za sterowanie aplikacją za pomocą wyjątków? 🤔

Reply:
Tak samo jak pokazywanie komunikatu o błędnej komendzie, też jest używany wyjątek. Poza tym gdzieś wyłapywanie wyjątków musi być bo UniversalNumberParser.Parse wykorzystuje Convert.ChangeType, które albo zwraca wartość albo wyrzuca wyjątek. C# nie ma czego takiego jak "TryChangeType".